app/settings.py

```diff
@@ -62,11 +62,25 @@
 DEBUG = env.DEBUG
 SILKY_PYTHON_PROFILER = env.SILKY_PYTHON_PROFILER
 
-USER_REFERENCE_COOKIE = env.OSF_AUTH_COOKIE_NAME
+OSF_AUTH_COOKIE_NAME = env.OSF_AUTH_COOKIE_NAME
 OSF_BASE_URL = env.OSF_BASE_URL.rstrip("/")
 OSF_API_BASE_URL = env.OSF_API_BASE_URL.rstrip("/")
 ALLOWED_RESOURCE_URI_PREFIXES = {OSF_BASE_URL}
+SESSION_ENGINE = "django.contrib.sessions.backends.cache"
+SESSION_COOKIE_NAME = env.OSF_AUTH_COOKIE_NAME
 SESSION_COOKIE_DOMAIN = env.SESSION_COOKIE_DOMAIN
+SESSION_COOKIE_SECURE = env.SESSION_COOKIE_SECURE
+SESSION_COOKIE_HTTPONLY = env.SESSION_COOKIE_HTTPONLY
+SESSION_COOKIE_SAMESITE = env.SESSION_COOKIE_SAMESITE
+OSF_AUTH_COOKIE_SECRET = env.OSF_AUTH_COOKIE_SECRET
+REDIS_HOST = env.REDIS_HOST
```

Comment:
GV Deployment notes: `REDIS_HOST` should be the same as what OSF uses. This maybe different for each server.